performance_metrics/include/performance_metrics/events_logger.hpp

```diff
@@ -45,13 +45,17 @@ class EventsLogger
 
   EventsLogger() = delete;
 
-  explicit EventsLogger(const std::string & filename);
+  explicit EventsLogger(const std::string & filename, const bool csv_out = false);
```

Comment:
```suggestion
  EventsLogger(const std::string & filename, bool csv_out = false);
```

Remove `explicit` (it should be used only when constructors have 1 argument) and `const` (data is copied, so `const` is noise)